app/models/katello/trace_status.rb

```diff
@@ -35,9 +35,12 @@ def to_global(_options = {})
     end
 
     def to_status(_options = {})
-      if host.host_traces.reboot_required.any?
+      traces = host.host_traces.pluck(:app_type).uniq
+      traces.delete('session')
+
+      if traces.include?('static')
```

Comment:
I'd prefer to store `static` in a constant (`Katello::HostTracer::APP_TYPE_REBOOT_REQUIRED`) and also use it in the `HostTracer` model itself.